PRs.md

```diff
@@ -0,0 +1,30 @@
+Pull Requests
+-------------
+
+**A good PR:**
+ - has a description; we are humans, we understand text better than code; a good description features context, what has been done and caveats of the chosen approach
+ - has no or minimal accidental changes - please open a separate PR, both PRs will be merged faster while much easier to review _one thing_ at a time
+ - has comments on the tricky / controversial parts of the code to guide reviewers
+ - mechanical refactoring should always be a separate PR and should be explained as such
```

Comment:
@antonrizov-tide noted that it will be very useful to have the description in the merge commit message. Should add.

Reply:
Done.